commands/login/index.js

```diff
@@ -1,28 +1,28 @@
 const error = require('error/typed');
 const p = require('commander');
-const program = new p.Command('losant configure');
+const program = new p.Command('losant login'); // TODO why was that like that?
 const getApi = require('../../lib/get-api');
 const retryP = require('../../lib/retryP');
 const c = require('chalk');
 const {
   saveUserConfig, logError, logResult
 } = require('../../lib/utils');
 const inquirer = require('inquirer');
+const { curry } = require('omnibelt');
 
-const signIn = async (isRetry) => {
+const signIn = curry(async (email, isRetry) => {
```

Comment:
I'm not sure that I would retain the email - cause what if they typoed their email address and *that* is why it failed?

Reply:
Should we do some sort of confirm? once we get to that step and then start over if they need to enter a new email?

Reply:
well, why not go back to the way it was where you request both again? was there a reason you stopped that?

Reply:
yeah that seemed really strange cause given the scenario that a user is NOT SSO. It would look like this the first time around.
``` 
Please enter your email:
Please enter your email: 
Please enter your password: 
```

Reply:
that works for me.